tools/test.py

```diff
@@ -21,13 +21,15 @@ def single_gpu_test(model, data_loader, show=False):
     results = []
     dataset = data_loader.dataset
     prog_bar = mmcv.ProgressBar(len(dataset))
+    score_thr = model.module.test_cfg['score_thr']
```

Comment:
`score_thr` in the test config is usually too small (e.g., 0.05) for visualization, which is usually set to `0.5`.